README.rst

```diff
@@ -184,7 +184,7 @@ You can find a quick demo of coala here:
 
 |asciicast|
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
+.. |asciicast| image:: https://raw.githubusercontent.com/coala/artwork/master/artwork/asciinema_demo/asciinema_demo.png
```

Comment:
Ok so I need to change the file source again to https://rawgit.com/coala/artwork/master/artwork/asciinema_demo/asciinema_demo.png
This would suffice?